include/swift/Runtime/EnvironmentVariables.h

```diff
@@ -24,6 +24,10 @@ void initialize(void *);
 
 extern swift::once_t initializeToken;
 
+// Define a typedef "string" in swift::runtime::environment to make string
+// environment variables work
+using string = const char *;
```

Comment:
wat

Reply:
Yeah, I know. It's because someone went mad with macros with the environment variable stuff. I'd like to fix that to use fewer macros and a bit more C++, which would remove the need for this horrifying line of code, but that's too big a thing to do as part of another piece of work.